drivers/ovsconstants.go

```diff
@@ -0,0 +1,43 @@
+package drivers
+
+const (
+	operCreateBridge oper = iota
+	operDeleteBridge
+	operCreatePort
+	operDeletePort
+)
+
+const (
```

Comment:
May be we should merge all these consts into one block? I am not sure if there is any golang specific convention here though.